src/lesson_5/script.js

```diff
@@ -0,0 +1,85 @@
+for (let i = 10; i <= 25; i +=0.5) {
+    console.log("i is equal to =", i);
+}
+
+const userNumber = prompt("Напишіть будь яке число");
+console.log("userNumber =", userNumber);
+
+let isPrime = "prime"; 
+
+for (var i = 2; i < userNumber; i++) {
+    if (userNumber % 1 === 0) {
```

Comment:
Here you need or change the condition to 
```suggestion
    if (userNumber % 2 === 0) {
```